SignallingWebServer/platform_scripts/cmd/common.bat

```diff
@@ -216,7 +237,7 @@ call :SetupCoturn
 exit /b
 
 :SetPublicIP
-FOR /f %%A IN ('powershell -command "(Invoke-Webrequest "http://api.ipify.org").content"') DO set PUBLIC_IP=%%A
+FOR /f %%A IN ('curl http://api.ipify.org') DO set PUBLIC_IP=%%A
```

Comment:
I did not realise windows now includes curl by default.

Reply:
Yeah, is cool! https://curl.se/windows/microsoft.html